DEVELOPER_CHEATSHEET.md

```diff
@@ -0,0 +1,19 @@
+# Quick reference to common development tasks
+
+## ClickHouse debugging
+
+### How do I see which queries are sent to ClickHouse
+
+In tests, run:
+
+```
+pytest <path-to-test> --log-cli-level=DEBUG
```

Comment:
You can pass arguments to bin/tests as well

Reply:
Yup, I typically don't use tbh but have added to https://github.com/PostHog/posthog.com/pull/2148 instead. I'll close this PR